### PR TITLE
PP-343: Fix PBS Build error on Windows

### PR DESCRIPTION
**Pull request by gajendrasharmagit**
**Issue-ID**
PP-343

**Problem description**
Fix PBS Build error on Windows

**Cause / Analysis**
pbs_version.h was unavailable.

**Solution description**
Generate pbs_version.h from pbs_version.h.in using VS pre-build event.

Build logs are attached to Jira to indicate the successful build.
**Checklist:**

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [x] I have added tests to cover my changes. To create automated tests, see **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**.
- [x] All new and existing automated tests have passed. See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**.
- [x] I have attached automated (PTL)/manual test logs to the associated Jira ticket.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: win_configure/projects.VS2008/gen_pbs_version.bat [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@echo off
+
+REM this genereates the pbs_version.h file.
+
+if exist "..\..\src\include\pbs_version.h.in" (
+COPY "..\..\src\include\pbs_version.h.in" "..\..\src\include\pbs_version.h" 
+)
```
Comment (subhasisb) [inline win_configure/projects.VS2008/gen_pbs_version.bat]: pbs_version.h.in is the file that configure will use as input to generate pbs_version.h. The idea is to use a staging machine to make a distributable source tar from the github sources, which will do the autotools step and create a complete tar source package for compilation on platforms that do not support autotools.

So, I dont think we should be copying pbs_version.h.in directly to pbs_version.h

**Comment by subhasisb** [comment]
Looks good - I sign off.

**Comment by subhasisb** [comment]
@gajendrasharmagit you need to attach test steps and also attach logs as evidence of testing done on the code. You need to attach them to the jira ticket as attachments (please dont add as comment, they get buried)

Once you complete the above and secure 2 sign offs, please rebase your branch so that your pull request is up to date with base.

**Comment by chitrac** [comment]
@gajendrasharmagit I sign off on the changes.
Thanks

**Comment by subhasisb** [comment]
All requirements complete, merging.

**Pull request merged**